src/cli/cli.cpp

```diff
@@ -5793,6 +5795,14 @@ template <> otError Interpreter::Process<Cmd("radiofilter")>(Arg aArgs[])
 }
 #endif
 
+template <> otError Interpreter::Process<Cmd("randombootid")>(Arg aArgs[])
+{
+    static const uint32_t bootid = otRandomNonCryptoGetUint32();
```

Comment:
How about using a member variable in `Interpreter` (and init it from constructor) instead of `static` variable?